src/FsCheck/Gen.fs

```diff
@@ -313,6 +313,19 @@ module Gen =
     [<CompiledName("ListOf")>]
     let listOfLength n arb = sequence [ for _ in 1..n -> arb ]
 
+    ///Generates a random permutation of the given sequence.
+    //[category: Creating generators]
+    [<CompiledName("Shuffle")>]
+    let shuffle xs =
+        gen {
+            let g = choose (-922337203, 922337203)
```

Comment:
If a person comes by this code later on, and attempts to understand it, and has access to only the code, but not this discussion thread, would he or she be any wiser?